docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -98,15 +98,41 @@ If you do not specify either, then the DaemonSet controller will create Pods on
 
 ## How Daemon Pods are Scheduled
 
-Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler.  However, Pods
+### Scheduled by DaemonSet controller (default)
+
+Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler. However, Pods
 created by the DaemonSet controller have the machine already selected (`.spec.nodeName` is specified
-when the Pod is created, so it is ignored by the scheduler).  Therefore:
+when the Pod is created, so it is ignored by the scheduler). Therefore:
 
  - The [`unschedulable`](/docs/admin/node/#manual-node-administration) field of a node is not respected
    by the DaemonSet controller.
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+In version 1.10, `ScheduleDaemonSetPods` was introduced as an alpha feature. This features schedules
+ DaemonSet Pods with the default scheduler instead of with the Daemonset controller.
+
+When this feature is enabled, the `NodeAffinity` term is added to the DaemonSet Pods, as in the following example.
+This enables the default scheduler to bind the Pod to the target host.
+
+```
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+    - matchExpressions:
+      - key: kubernetes.io/hostname
+        operator: In
+        values:
+        - target-host-name
+```
+
+If the `kubernetes.io/hostname` affinity exists, `.pod.Spec` is updated but no changes are made to
+the `spec.template` of the DaemonSet.
```

Comment:
It would be desirable to describe the side effects of such an "intermediate" state.
Do you mean the Pod scheduling still works as expected although we cannot get the precise spec of the DaemonSet when doing a `kubectl get daemonsets/<foo> -o yaml` ?

Reply:
no change to scheduler, it just follow current behaviour. just highlight the pod of DaemonSet maybe not exactly same with DamonSet's spec.template.

Reply:
This sounds an unexpected behavior. We'd better use a callout for this.

Reply:
what kind of 'callout' do you mean?

Reply:
I mean something like the following:

```
**Note** <Here are something users should be aware of ... i.e, your text>
{: .note}
```